src/EnergyPlus/DXCoils.cc

```diff
@@ -15706,11 +15706,10 @@ void GetDXCoilIndex(EnergyPlusData &state,
 
     DXCoilIndex = UtilityRoutines::FindItemInList(DXCoilName, state.dataDXCoils->DXCoil);
     if (DXCoilIndex == 0) {
-        if (present(SuppressWarning)) {
+        if (!SuppressWarning) {
             //     No warning printed if only searching for the existence of a DX Coil
-        } else {
-            if (present(ThisObjectType)) {
-                ShowSevereError(state, ThisObjectType + ", GetDXCoilIndex: DX Coil not found=" + DXCoilName);
+            if (!ThisObjectType.empty()) {
+                ShowSevereError(state, fmt::format("{}, GetDXCoilIndex: DX Coil not found={}", ThisObjectType, DXCoilName));
```

Comment:
And finally the string handling is changed to something compatible with `string_view`.